sale_order_type/tests/test_sale_order_type.py

```diff
@@ -218,3 +231,15 @@ def test_sale_order_quotation_validity(self):
         order = self.create_sale_order()
         order.onchange_type_id()
         self.assertEqual(fields.Date.to_string(order.validity_date), "2022-01-11")
+
+    def test_sale_copy_function(self):
+        """
+        Test when duplicating the sale order the account analytic account is set.
+        """
```

Comment:
PEP257 indicates that the docstring should be put this way:

```suggestion
        """Test when duplicating the sale order the account analytic account is set.
        """
```